source/compatibility.tex

```diff
@@ -2794,7 +2794,7 @@
 \pnum
 For compatibility with the C standard library\indextext{library!C standard},
 the \Cpp{} standard library provides the C headers enumerated
-in~\ref{depr.c.headers}, but their use is deprecated in \Cpp{}.
+in~\ref{support.c.headers}, but their use is deprecated in \Cpp{}.
```

Comment:
Since we move the clause out of Appendix D, is the use of C headers still deprecated? In the paper it seems that the author intended to make the C headers no longer deprecated.